inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -217,11 +217,22 @@ def is_experimental_paper(obj, eng):
 @with_debug_logging
 def is_arxiv_paper(obj, *args, **kwargs):
     """Check if the record is from arXiv."""
-    arxiv_id = get_arxiv_id(obj.data)
-    categories = get_value(obj.data, 'arxiv_eprints.categories')
 
-    if arxiv_id or categories:
-        return True
+    acquisition_source = obj.data.get('acquisition_source')
+
+    if acquisition_source:
+        method = acquisition_source.get('method')
```

Comment:
```
method = get_value(obj.data, 'acquisition_source.method')
```

Reply:
since I already have `acquisition_source` and it's an object, I don't see any point to call `get_value`

Reply:
you can replace 3 lines by one

Reply:
ah in that case, got it, I guess this function never raises exceptions but returns `None`,

Reply:
still I will need another nested access  in `elif`, for `source`

Reply:
good point, nevermind then.